backend/python/app/services/implementations/story_service.py

```diff
@@ -97,36 +97,43 @@ def create_translation(self, entity):
 
     def get_story_translations(self, user_id, translator, language, level):
         try:
-            return (
-                db.session.query(
-                    Story.id.label("story_id"),
-                    Story.title.label("title"),
-                    Story.description.label("description"),
-                    Story.youtube_link.label("youtube_link"),
-                    Story.level.label("level"),
-                    StoryTranslation.id.label("id"),
-                    StoryTranslation.language.label("language"),
-                    StoryTranslation.stage.label("stage"),
-                    StoryTranslation.translator_id.label("translator_id"),
-                    StoryTranslation.reviewer_id.label("reviewer_id"),
+            stories = (
```

Comment:
I like what you did for `get_story_translation` to use just 1 query - what happens if you use a similar approach here?

Reply:
It should be doable but would require a lot of post-processing since it would entail combining all the storytranslationcontent rows with the same `story_translation_id` together into a dict and doing this for all rows

Reply:
understood, happy with this state then 👍